packages/dynamics-lib/src/queries/__tests__/activity.queries.spec.js

```diff
@@ -69,4 +69,63 @@ describe('Activity Service', () => {
       })
     })
   })
+
+  describe('updateActivity', () => {
+    const mockResponse = {
+      '@odata.context': 'https://dynamics.om/api/data/v9.1/defra_UpdateRCRActivityResponse',
+      ReturnStatus: 'success',
+      SuccessMessage: 'RCR Activity - updated successfully',
+      ErrorMessage: null,
+      oDataContext: 'https://dynamics.com/api/data/v9.1/defra_UpdateRCRActivityResponse'
+    }
+
+    const errorResponse = {
+      '@odata.context': 'https://dynamics.om/api/data/v9.1/defra_UpdateRCRActivityResponse',
+      RCRActivityId: null,
+      ReturnStatus: 'error',
+      SuccessMessage: '',
+      ErrorMessage: 'Failed to update activity',
+      oDataContext: 'https://dynamics.com/api/data/v9.1/defra_UpdateRCRActivityResponse'
+    }
```

Comment:
Please can you convert both of these from variables to functions, so that a fresh copy of the data is obtained each time? Also, change the name to `getMockResponse` / `getErrorResponse`